askup/views.py

```diff
@@ -26,7 +32,12 @@ def get_queryset(self):
 
         Overriding the get_queryset of generic.ListView
         """
-        return Qset.objects.filter(parent_qset=None).order_by('name')
+        user = self.request.user
+
+        if user.is_authenticated():
+            return Qset.objects.filter(parent_qset=None, users__id=user.id).order_by('name')
```

Comment:
Why do we filtering by users__id? It seems that we want to show all organization's qsets, don't we?